src/main/java/seedu/tassist/model/person/LabScore.java

```diff
@@ -0,0 +1,63 @@
+package seedu.tassist.model.person;
+
+import seedu.tassist.logic.commands.UpdateLabScoreCommand;
+import seedu.tassist.logic.commands.exceptions.CommandException;
+
+/**
+ * Holds the score and max score for a lab.
+ */
+public class LabScore {
+    private int labScore;
+    private int maxLabScore = 25;
```

Comment:
Max lab score seems to be a constant, perhaps
```suggestion
    private static int MAX_LAB_SCORE = 25;
```